api/src/main/java/edu/cornell/mannlib/vitro/webapp/services/shortview/FakeApplicationOntologyService.java

```diff
@@ -494,7 +494,10 @@ public FakeVivoPeopleDataGetter(VitroRequest vreq, String individualUri) {
 			this.individualUri = individualUri;
 			this.vreq = vreq;
 			this.ctx = vreq.getSession().getServletContext();
-			this.langCtx =  vreq.getLocale().getLanguage() + "-"+vreq.getLocale().getCountry(); // UQAM-Optimization add the linguistic context
+			this.langCtx =  vreq.getLocale().getLanguage(); // UQAM-Optimization add the linguistic context
+            if (!vreq.getLocale().getCountry().isEmpty()) {
```

Comment:
The formatting in the VIVO codebase is... inconsistent. However, it is helpful not to exacerbate the chaos. Would you mind changing the indentation of the lines you added to either use spaces or tabs in order to match the lines around it?

Reply:
i changed it.